nrm_django/nrm_site/settings/eb.py

```diff
@@ -16,4 +17,12 @@ def database_dict_from_rds_env():
 
 
 DATABASES = {"default": database_dict_from_rds_env()}
-ALLOWED_HOSTS = ["development.eba-ny2imkrt.us-east-2.elasticbeanstalk.com"]
+ALLOWED_HOSTS = [
+    "development.eba-ny2imkrt.us-east-2.elasticbeanstalk.com",
+    "ga-development.eba-c5v3ebxp.us-gov-west-1.elasticbeanstalk.com",
+]
+try:
+    EC2_IP = requests.get("http://169.254.169.254/latest/meta-data/local-ipv4").text
+    ALLOWED_HOSTS.append(EC2_IP)
+except requests.exceptions.RequestException:
+    pass
```

Comment:
I'm trying to think through how passing this exception will work. If we can't get EC2_IP and append it to ALLOWED_HOSTS, won't Django blow up anyway with an ALLOWED_HOSTS error? Do we want to be suppressing this, or should we want to blow up here? Or am I just not looking at it the right way?

Reply:
I think the local IP is only needed to make health checks succeed. The app will still work because we unconditionally add the inbound load balancer on line 20. I think that "best effort" is reasonable here and we can do nothing if the request fails for some reason.